way_to_home/notification/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the notification entity."""
+
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+from way.models import Way
+
+
+WEEK = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
```

Comment:
You should start a week from Sunday

Reply:
Was basing it on python datetime standarts 
![screenshot from 2018-12-14 13-51-11](https://user-images.githubusercontent.com/39067428/50002035-7b6a8580-ffa7-11e8-93ba-c16ea1efe53e.png)

Reply:
remove this line